source/includes/options-mongodrdl.yaml

```diff
@@ -279,16 +279,30 @@ description: |
   .. versionadded:: 2.12
 
   MongoDB URI connection string.
-optional: true
```

Comment:
`--uri` is actually still optional! A user can do either:
`mongodrdl --uri 'mongodb://...'`, or
`mongodrdl --db ... --port ... ...`

Reply:
Actually the "optional" directive there doesn't do anything. There's no "optional" field in this .rst construction. The fact that it exists in this file at all is a relic of some ancient copy-paste business from the server manual. One of these days I'll remove all the "optional" directives from all the .yaml files in this whole repository.